### ISSUE TITLE
[Doc error]: Outdated doc for LLAMA

### ISSUE THREAD

**Issue by llCurious** [young-geng/EasyLM#59]
I try to run LLAMA using EasyLM. I follow the [README](https://github.com/young-geng/EasyLM/blob/main/docs/llama.md) for llama. The first step is conver raw LLAMA parameters.

``` sh
python -m EasyLM.models.llama.convert_torch_to_easylm.py \
    --checkpoint_dir='path/to/torch/llama/checkpoint' \
    --output_dir='path/to/output/easylm/checkpoint' \
    --streaming=True
```

The arg output_dir does not appear in `convert_torch_to_easylm.py`, which should be `output_file` now, as shown in [code](https://github.com/young-geng/EasyLM/blob/main/EasyLM/models/llama/convert_torch_to_easylm.py#L17-L21).

I wonder if the doc is outdated?

**Comment by young-geng** [comment]
Yeah the doc is a bit outdated due to a lot of changes I made recently. I will try to update it more frequently.
